src/js/components/explorer/landing/ExplorerLanding.jsx

```diff
@@ -6,6 +6,8 @@
 import React from 'react';
 import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
 
+import * as Icons from 'components/sharedComponents/icons/Icons';
```

Comment:
If we're just importing one icon, we can make this import lighter weight by only importing the icon rather than all of them:
```
import { Glossary } from 'components/sharedComponents/icons/Icons';

....


<Glossary />
```